src/frontend/src/error.rs

```diff
@@ -271,12 +271,6 @@ pub enum Error {
         source: common_grpc_expr::error::Error,
     },
 
-    #[snafu(display("Failed to deserialize insert batching: {}", source))]
```

Comment:
Looks like we can also delete these variants:
- `BumpTableId`
- `CreateDatabase`
- `CreateTableOnInsertion`
- `CatalogManager`
- `FullTableName`
- `ExecuteSql`
- `VectorComputation`